src/unix/linux-core.c

```diff
@@ -220,6 +220,7 @@ void uv__io_poll(uv_loop_t* loop, int timeout) {
   int i;
   int user_timeout;
   int reset_timeout;
+  user_timeout = 0;
```

Comment:
Sorry, I could have sworn I left this comment when I left my review on the PR... could you move this assignment down to the `else` block around line 284.

Reply:
Sorry, I just noticed this. I'll move it and update shortly. Thanks